### DISCUSSION TITLE
Satellaview Cart Flashing

### DISCUSSION THREAD

**Discussion by Bandmo** [sanni/cartreader#362] [Need help]
Hey there. Is there a way I can load a game onto the BSX cart with memory card? If so, how would I go about doing this? Thanks!

**Comment by sanni** [comment]
Yes that is possible. You need to select "Super Nintendo" then "Satellaview BS-X" and then "Write Memory Pack".
For hardware you need the BS-X cartridge(SHVC-ZBSJ-JPN), a game cartridge like Same Game is not supported, and an 8M Memory Pack.

But dump the contents of the Pack first before overwriting it, even if it seems empty, you never know what you might find.

**Comment by Bandmo** [comment]
Okay. I have all of that. What format do the games need to be for the memory pack?

- Brendon

**Comment by sanni** [comment]
Unheadered raw dump like in no-intro satellaview set.

**Comment by Bandmo** [comment]
Thank you! I found some. I tried loading one up, and it still didn't find a
game when I tried to play it. I realize that I never finished building the
snes pcb for the HW4, and don't have the supercic installed yet. Will that
be required in order to dump or flash?

**Comment by sanni** [comment]
Does it say "Finished successfully" at the end of the write?

I know it needs the clock generator but I don't think it needs the snesCIC but I never actually tested it without and I don't have any SV stuff anymore to confirm it.

Here is an old video maybe you can compare it and see if flashing differs for you at any point:
https://www.youtube.com/watch?v=82fBadcm1bY

Also, if you dump the 8M Pack again and then compare the dump to the original file, does it match? You can use the compare function of an hex editor for that.

**Comment by Bandmo** [comment]
That all looks very similar, except for the part when he writes the FF.Sram file in his “test” folder  Is that something I need to do as well?

Thanks again. 

- Brendon

**Comment by sanni** [comment]
I doubt it but only one way to find out I guess. It's just a save file filled with 0xFF to erase the SRAM chip.

[FF.zip](https://github.com/sanni/cartreader/files/8622318/FF.zip)
(unzip before flashing)

**Comment by Bandmo** [comment]
Awesome! I found the problem. The file titles were too long. They were not showing up on the screen when choosing the file to flash. Now, is there a way to flash multiple games at one time so there are more than one on the in-game menu at a time? Otherwise the cart gets erased each time a rom is flashed on it.

**Comment by sanni** [comment]
If you search google for "Satellaview 8M Dump (Picross, SatellaQ, Machi Magazine)" you'll find an 1MB Satellaview file with 3 games in it. You can try to flash it and see if all three games can be started. If yes then someone just has to find out how to make a 1MB Satellaview file with multiple games from scratch.

**Comment by Bandmo** [comment]
So the flashing is really spotty. Sometimes it flashes, most times not. I wonder if maybe I messed up the pins in my snes cart when I inserted a wonderswan pcb that was too thick. I have an extra slot I’m going to replace it with and see if it works better. Ill also check out the rom combining. Thanks.

**Comment by Bandmo** [comment]
So I replaced the slot. What games can be flashed is still really spotty. I can do the 8m dump with picross, a zelda game, and what looks like a Japanese takeout menu. But I really cant get many more things to flash. Any thoughts??
